integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
@duglin 

Of cause I did use your pasted codes but still can't get 'out', its still blank as I
firstly tried buildImageWithStdoutStderr.